src/data-interfaces/__mocks__/HoloFuelInterface.js

```diff
@@ -1,14 +1,18 @@
 const currentDataTimeIso = () => new Date().toISOString()
 
-const successfulTransactionResponse = ({ transactionId, amount, counterparty, status, type }) => {
+const successfulTransactionResponse = ({ transactionId, amount, counterparty, status, type, fees, presentBalance, notes }) => {
   return {
     id: transactionId,
     amount: amount || 0,
     counterparty: counterparty || 'Data not avail until next refetch, retrieve from gql cache...',
     direction: type === 'offer' ? 'outgoing' : 'incoming',
     status,
     type,
-    timestamp: currentDataTimeIso
+    timestamp: currentDataTimeIso,
+    // NOTE: the following details are ONLY available in the 'completed transactions'...
+    fees: fees || 0,
```

Comment:
we typically don't want to provide defaults for these as they will overwrite the cache. Can explain more on our call today.